Source/MagneX.cpp

```diff
@@ -140,6 +142,8 @@ void InitializeMagneXNamespace() {
     pp.get("timedependent_alpha",timedependent_alpha);
 
     pp.get("demag_coupling",demag_coupling);
+
+    pp.get("FFT_solver",FFT_solver);
```

Comment:
I would wrap this in 

if (demag_coupling==1) {

}

since you don't need to force the user to select an FFT_solver if not using demag.